app/gui2/src/stores/project/index.ts

```diff
@@ -506,7 +506,7 @@ export const useProjectStore = defineStore('project', () => {
   watchEffect((onCleanup) => {
     // For now, let's assume that the websocket server is running on the same host as the web server.
     // Eventually, we can make this configurable, or even runtime variable.
-    const socketUrl = new URL(location.origin)
+    const socketUrl = new URL('http://localhost:1234')
```

Comment:
I haven't managed to open a project. This is a message I found in the log:
```
[INFO] [2024-04-20T08:05:38+02:00] [org.enso.languageserver.boot.resource.RepoInitialization] Initializing suggestions repo [org.enso.searcher.sql.SqlDatabase@3d4d1e81]...
[WARN] [2024-04-20T08:05:38+02:00] [org.enso.editions.updater.EditionUpdater] Failed to fetch editions from [http://localhost:1273/editions/]: {}
org.enso.downloader.http.HTTPException: Server responded with: [java.net.ConnectException].
        at org.enso.runtime/org.enso.downloader.http.HTTPException$.apply(HTTPException.scala:9)
        at org.enso.runtime/org.enso.downloader.http.HTTPDownload$.$anonfun$fetchString$2(HTTPDownload.scala:71)
```

Reply:
I even got debugging working with:
```bash
enso$ POLYGLOT_YDOC_SERVER_DEBUG=true npm --workspace=enso-gui2 run dev
ydoc: Debugger listening on ws://127.0.0.1:34567/HwpxX-S2_c-dB0qRAMhY-LmSWALU7RSvXF5IBRkgeoA
For help, see: https://www.graalvm.org/tools/chrome-debugger
ydoc: E.g. in Chrome open: devtools://devtools/bundled/js_app.html?ws=127.0.0.1:34567/HwpxX-S2_c-dB0qRAMhY-LmSWALU7RSvXF5IBRkgeoA
```
I can connect to your process via ChromeDevTools:

![obrazek](https://github.com/enso-org/enso/assets/26887752/584fa8a8-2910-4eb3-a3d6-9757faa218da)

Reply:
No, it's not related